scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -219,45 +219,31 @@ if [[ "${RUN_TESTS}" == "false" ]]; then
     fi
 fi
 
-if [[ ${#ARGS} == 0 ]]; then
-    ARGS=("--with-coverage"
-          "--cover-erase"
-          "--cover-html"
-          "--cover-package=airflow"
-          "--cover-html-dir=airflow/www/static/coverage"
-          "--with-ignore-docstrings"
-          "--rednose"
-          "--with-xunit"
-          "--xunit-file=${XUNIT_FILE}"
-          "--with-timer"
```

Comment:
The pytest "equivalent" of this is `--durations=100` (to show timing for slowest 100 tests). Shall we add it back?

Reply:
Done